spec/controllers/charts_controller_spec.rb

```diff
@@ -7,6 +7,18 @@
   let(:invalid_attributes) {
     skip('Add a hash of attributes invalid for your model')
   }
+  describe "GET #index" do
+    it 'renders the index-template' do
+      get :index
+      expect(response).to be_success
+    end
+    it "assigns all charts as @charts" do
+      (1..5).each { |num| Chart.new(id: num, title: 'fabrication') }      
+      charts = Chart.all
```

Comment:
I've been doing some searching but I could not find out how to create a collection of Model ( Chart::ActiveRecord_Relation) in Fabrication `ruby @charts = plenty of Fabrication(:chart)` do you have an idea?